django-stubs/contrib/admin/options.pyi

```diff
@@ -26,8 +30,11 @@ from django.db.models.fields import Field
 
 IS_POPUP_VAR: str
 TO_FIELD_VAR: str
-HORIZONTAL: Any
-VERTICAL: Any
+HORIZONTAL: Literal[1] = ...
+VERTICAL: Literal[2] = ...
+
+# TODO: Is this ok? Is this the way to mark a type as stub-only?
```

Comment:
@sobolevn What's your opinion here? Do you use stub-only types in other places? Is this consider OK?

Reply:
Yes, this is fine.